build/dependencies.props

```diff
@@ -2,9 +2,9 @@
   <PropertyGroup>
     <BenchmarkDotNetPackageVersion>0.11.3</BenchmarkDotNetPackageVersion>
     <GoogleProtobufPackageVersion>3.6.1</GoogleProtobufPackageVersion>
-    <GrpcCorePackageVersion>1.19.0-dev201902121041</GrpcCorePackageVersion>
-    <GrpcCoreApiPackageVersion>1.19.0-dev201902121041</GrpcCoreApiPackageVersion>
-    <GrpcToolsPackageVersion>1.19.0-dev201902121041</GrpcToolsPackageVersion>
+    <GrpcCorePackageVersion>1.19.0-pre1</GrpcCorePackageVersion>
+    <GrpcCoreApiPackageVersion>1.19.0-pre1</GrpcCoreApiPackageVersion>
+    <GrpcToolsPackageVersion>1.19.0-pre1</GrpcToolsPackageVersion>
     <MicrosoftAspNetCoreTestHostPackageVersion>3.0.0-preview-18579-0056</MicrosoftAspNetCoreTestHostPackageVersion>
     <MicrosoftExtensionsLoggingTestingPackageVersion>2.0.0-*</MicrosoftExtensionsLoggingTestingPackageVersion>
```

Comment:
I still think we should remove this dependency from a purity standpoint. Thoughts @JamesNK ?